src/scripts/component/master.js

```diff
@@ -57,11 +49,9 @@ let component = {
 
         utils.log('render component');
 
-        this.$element.find('[data-toggle="tooltip"]').tooltip();
-
         return this;
 
-    },
+    }
```

Comment:
since we removed the comma and updated to a Class, for style consistency I would suggest a new line here